src/Umbraco.Web.UI.Client/src/views/templates/edit.controller.js

```diff
@@ -457,7 +458,7 @@
                 return templateName;
 
             } else {
-                return "No master";
+                return "no master";
```

Comment:
@perploug I know this is for catching an error & unlikely to be seen from what I can tell in the code - but as you have gone to hard work of changing all of these hardcoded strings. Worth doing one more?